iis-wf/iis-wf-export-actionmanager/src/main/resources/eu/dnetlib/iis/wf/export/actionmanager/sequencefile/oozie_app/workflow.xml

```diff
@@ -237,7 +251,7 @@
                 <value>${action_set_id_document_similarities_standard}</value>
             </property>
             <property>
-                <name>export.action.setid.matched_doc_organizations</name>
+                <name>export.action.setid.document_affiliations</name>
```

Comment:
I had to introduce this fix to make exporter module work. 

Exporter composes property name holding action set identifier by concatenating `export.action.setid.` with `AlgorithmName` which in this case was named `AlgorithmName#document_affiliations`.